packages/stark-core/tslint.json

```diff
@@ -30,6 +30,63 @@
         }
       }
     ],
-    "use-view-encapsulation": false
+    "use-view-encapsulation": false,
+    "variable-name": [
+      true,
+      "check-format",
+      "require-const-for-all-caps",
+      "allow-leading-underscore",
+      "allow-trailing-underscore",
+      "ban-keywords"
+    ], // option "require-const-for-all-caps" added in 5.11
+    "no-unbound-method": [true, { "ignore-static": true, "whitelist": ["expect"], "allow-typeof": true, "allow-delete": true }], // "whitelist" added in 5.13, "allow-delete" added in 5.14
+    "file-name-casing": [true, { "public_api.ts": "snake-case", ".*": "kebab-case" }], // 5.11 snake-case option added in 5.12
+    "ban-ts-ignore": true, // 5.12
+    "function-constructor": true, // 5.12
+    //    "increment-decrement": true, // 5.12
+    //    "no-default-import": [true, { "fromModules":  "^someModule"}], // 5.12
+    "unnecessary-constructor": [true, { "check-super-calls": true }], // 5.12 and option added in 5.20
+    //    "comment-type": [true, "doc", "singleline"], // 5.12
+    "unnecessary-bind": true, // 5.12
+    //    "no-restricted-globals": [true, "name", "length", "event"], // 5.13
+    "no-tautology-expression": true, // 5.14
+    "static-this": true, // 5.14
+    "unnecessary-else": [true, { "allow-else-if": true }], // 5.14 option added in 5.16
+    //    "no-null-undefined-union": true, 5.15
+    "no-async-without-await": true, // 5.18
+    "strict-comparisons": [true, { "allow-object-equal-comparison": true, "allow-string-order-comparison": true }], // 5.18
+    //    "no-for-in": true, // 5.19
+    "invalid-void": true, // 5.19
+    //    "strict-string-expressions": true, // 5.19
+    "no-promise-as-boolean": true // 5.19
+    // TSLint rules that were previously on tslint:recommended prior to v6.0
+    //    "arrow-parens": true,
```

Comment:
I've added these rules that were removed from `tslint:recommended` as from TSLint v6.

Check PR https://github.com/NationalBankBelgium/code-style/pull/34 for the explanation about this and the discussion on what we should do about it.